jni_gen/lib/src/util/rename_conflict.dart

```diff
@@ -0,0 +1,54 @@
+String renameConflict(Map<String, int> counts, String name) {
+  name = kwRename(name);
+  if (counts.containsKey(name)) {
+    final count = counts[name]!;
+    final renamed = '${name}_$count';
+    counts[name] = count + 1;
+    return renamed;
+  }
+  counts[name] = 1;
+  return name;
+}
+
+String kwRename(String name) {
+  if (_keywords.contains(name)) {
+    return '${name}_';
+  }
+  return name;
+}
+
+// TODO: Add keywords(C) - keyword(Java)
+// TODO: change C constructor naming from 'new' to something else?
+const Set<String> _keywords = {
```

Comment:
Are these Dart's keywords? There's quite a few missing. Did you add Dart's keywords but then exclude Java keywords or something?

Reply:
Yes I was lazy and excluded Java's keywords because they cannot be in Java's identifier names.